### PR TITLE
Send form feedback

### PR DESCRIPTION
**Pull request by ryanwoldatwork**
the goal of this pr is to send an email to the user who archives a form, requesting product feedback

this pr also updates gems and enables additional controller specs

### CODE REVIEW

**Comment by ryanwoldatwork** [comment]
still working on the email text and accompanying form

File: app/views/user_mailer/form_feedback.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<p>
+    You recently archived the form, <%= @form.title %> in Touchpoints.
```
Comment (sanason) [inline app/views/user_mailer/form_feedback.html.erb:2]: I feel like there should be a comma after `@form.title` too.

File: app/views/user_mailer/form_feedback.text.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+You recently archived the form, <%= @form.title %> in Touchpoints.
+
+The form was created <%= time_ago_in_words(@form.created_at) %> ago.
+The form received <%= @form.response_count %> responses.
+
+Please take this <%= link_to "short survey", @feedback_url %> to provide feedback regarding the Touchpoints application and/or the Touchpoints team.
```
Comment (sanason) [inline app/views/user_mailer/form_feedback.text.erb:6]: Is it supposed to work if you put an anchor tag in the `text/plain` version?

In Mailcatcher, at least, it looks weird.
Screenshot 2024-06-11 at 2 39 45 PM

File: app/mailers/user_mailer.rb [hunk 3]
```diff
@@ -16,6 +16,14 @@ def notification(title: '', body: '', path: '', emails: [])
          to: emails
   end
 
+  def form_feedback(form_id:, email:)
+    set_logo
+    @form = Form.find(form_id)
```
Comment (sanason) [inline app/mailers/user_mailer.rb:21]: When a user submits the "About your experience using Touchpoints" form, will there be any way to tell which archived form they are submitting it about? Would it be useful to know?

**Review by sanason — COMMENTED** [review]
2 minor comments and 1 question.

File: app/views/user_mailer/form_feedback.html.erb (hunk 1 above)
Comment (ryanwoldatwork) [reply app/views/user_mailer/form_feedback.html.erb:2]: incorporated

File: app/views/user_mailer/form_feedback.text.erb (hunk 2 above)
Comment (ryanwoldatwork) [reply app/views/user_mailer/form_feedback.text.erb]: this has been addressed, only showing 'short survey' as text and relying on the URL below.

File: app/mailers/user_mailer.rb (hunk 3 above)
Comment (ryanwoldatwork) [reply app/mailers/user_mailer.rb:21]: good call. will add ?location_code

File: app/mailers/user_mailer.rb [hunk 4]
```diff
@@ -16,6 +16,14 @@ def notification(title: '', body: '', path: '', emails: [])
...
+    @feedback_url = "https://touchpoints.app.cloud.gov/touchpoints/522e395c?location_code=#{form.short_uuid}"
```
Comment (sanason) [inline app/mailers/user_mailer.rb:22]:
```suggestion
    @feedback_url = "https://touchpoints.app.cloud.gov/touchpoints/522e395c?location_code=#{@form.short_uuid}"
```

**Review by sanason — APPROVED** [review]
👍

**Pull request merged**